frontend/src/metabase/css/query_builder.module.css

```diff
@@ -496,22 +461,22 @@
   cursor: pointer;
 }
 
-.QuestionTooltipTarget:after {
+:global(.QuestionTooltipTarget:after) {
   content: "?";
   font-size: 13px;
   font-weight: bold;
 }
 
-.FilterInput:hover .FilterRemove-field {
+:global(.FilterInput:hover .FilterRemove-field) {
   opacity: 1;
 }
 
 /* need to do this ugliness to override the locally scoped font size from Scalar.css */
-.QueryBuilder .ScalarValue {
+:global(.QueryBuilder .ScalarValue) {
```

Comment:
You already removed this class `.QueryBuilder` from `frontend/src/metabase/query_builder/components/view/View.jsx`

So this one should be migrated.

#### This PR
![image](https://github.com/metabase/metabase/assets/1937582/a5d67659-37ed-4d81-b7d4-16eac3c27cf9)

#### `master`
![image](https://github.com/metabase/metabase/assets/1937582/7bddee82-12fb-43db-a2c9-5947fe3d1bab)